gudpy/gui/widgets/dialogs/nexus_processing_dialog.py

```diff
@@ -83,19 +83,24 @@ def initComponents(self):
             )
         ) as fp:
 
-            spectra = fp["/raw_data_1/detector_1/spectrum_index"][()][
-                :
-            ].tolist()
+            if fp.__contains__("/raw_data_1/detector_1/spectrum_index"):
```

Comment:
```suggestion
            if "/raw_data_1/detector_1/spectrum_index" in fp:
```
Should avoid calling double underscore 'dunder' methods externally! Using `in` calls `__contains__` but in a safer way.